tests/adjoint_born_test.py

```diff
@@ -31,29 +35,34 @@ def test_adjoint_born():
 
     tn = 1000.
     shape = (101, 101)
+    nbl = 40
     model = demo_model('layers-isotropic', origin=(0., 0.), shape=shape,
-                       spacing=(10., 10.), nbl=40, nlayers=5)
+                       spacing=(10., 10.), nbl=nbl, nlayers=2,
+                       space_order=4)
     model0 = demo_model('layers-isotropic', origin=(0., 0.), shape=shape,
-                        spacing=(10., 10.), nbl=40, nlayers=5)
-    nb = model.nbl
-    model0.vp = ndimage.gaussian_filter(model0.vp.data[nb:-nb, nb:-nb],
-                                        sigma=(1, 1), order=0)
+                        spacing=(10., 10.), nbl=nbl, nlayers=2,
+                        space_order=4)
+
+    gaussian_smooth(model0.vp, sigma=(1, 1))
     geometry0 = setup_geometry(model0, tn)
 
     # Pure Devito
-    solver0 = AcousticWaveSolver(model0, geometry0, space_order=8)
-    dm = (model.vp.data**(-2) - model0.vp.data**(-2))
+    solver0 = AcousticWaveSolver(model0, geometry0, space_order=4)
+    dm = model.vp.data**(-2) - model0.vp.data**(-2)
 
-    grad_devito = np.array(solver0.born(-dm)[0].data)
+    grad_devito = np.array(solver0.jacobian(-dm)[0].data)
```

Comment:
would put `dm = model0.vp.data**(-2) - model.vp.data**(-2)` and `jacobian(dm)` toavoid that minus sign, may make people think that gradient sign is wrong.

Reply:
Alright, that is more reasonable.

Reply:
Regardless of this, we want to compare linearized data w/ the following gradient (computed via pytorch) at `d=0`:

<img src="https://render.githubusercontent.com/render/math?math=\nabla_d \ \frac{1}{2} \left \| \mathbf{J}^T \mathbf{d} - \delta \mathbf{m} \right \|^2_2 = - \mathbf{J} \delta \mathbf{m} ">

We need a minus sign somewhere.

Reply:
That's an original gradient :D  but yes makes sense then. 
Maybe add a comment or docstring for it? Feels weird to see that minus sign without it? but gtg

Reply:
Yeah it is a gradient just for the sake of testing. I moved that minus sign to few lines later with a comment stating that we need the negative gradient of the loss function to get linearized data. 

Thanks will merge.